### PR TITLE
Meta butchering

### PR DESCRIPTION
**Pull request by int-e**
Changes on top of #5... feel free to squash them into your single commit.

### CODE REVIEW

**Comment by int-e** [comment]
I've made sure that the reported usage is not changed:
```
usage: nsapass get [-h] [-a] [-z] [USERNAME] [INDEX]
usage: nsapass add [-h] [-a] [-z] [-g LEN SET | -m] [-u USERNAME] [-i INDEX] [-n NOTE]
usage: nsapass del [-h] [-z] [USERNAME] [INDEX]
usage: nsapass undel [-h] [-z] [USERNAME] [INDEX]
usage: nsapass mod [-h] [-z] [-g LEN SET | -m] [-u USERNAME] [-i INDEX] [-n NOTE] [USERNAME] [INDEX]
usage: nsapass view [-h] [-a] [-z]
usage: nsapass chpass [-h]
```
Note though that `get -a` now actually has an effect: `get` ignores archived entries by default, and it ignores non-archived entries if `-a` is given.

File: nsapass [hunk 1]
```diff
@@ -245,7 +245,7 @@ def checkquery(db, username, index, arc=None):
     if username is None:
         if len(usernames) == 1:
             printwarn('only "{}" in db.  '
-                "you must've wanted him.  got it!".format(usernames[0]))
+                "you must've wanted this one.  got it!".format(usernames[0]))
```
Comment (int-e) [inline nsapass:248]: (avoid gender confusion)

**Comment by al-caveman** [comment]
oo.  thanks.  

i like the stricter enforcement of `-a`.  it made me think:  perhaps we should have a policy that the archive is only accessible by any instruction if-and-only-if `-a` is present, and the non-archive is accessible by any instruction if-and-only-if `-a` is absent.  this way i may not accidently delete a real entry thinking it was an archived one.

**Comment by int-e** [comment]
Yes, I think adding `-a` to other commands (namely `del` and `mod`, but not `undel`, which implies `-a` now--sorry, forgot to mention that) makes sense. I even considered doing that, but I wanted to see whether you like the change for `add` first. It shouldn't be hard anyway... adding `-a` to the respective commands and passing the flag on to `checkquery` should suffice.

**Pull request merged**